src/Cipher/Autokey/AutokeyStream.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { Button, ButtonGroup, ButtonToolbar, FormControl, ToggleButton, ToggleButtonGroup } from 'react-bootstrap';
+import LocalStorage from 'Data/LocalStorage';
+import { AutoKeyString } from 'puzzle-lib';
```

Comment:
@kfarnung should I bump the version number of puzzle-lib to indicate the new API is available and required?

Reply:
If you run `npm install puzzle-lib` it should update the file for you.